products/ha_memgraph/v0.15.0/faq.md

```diff
@@ -0,0 +1,86 @@
+# Frequently Asked Questions
+
+1. ##### What is HA Memgraph?
+
+HA Memgraph is an ACID compliant, highly available, transactional in-memory
+graph database management system based on the Raft protocol.
+
+2. ##### Does being in-memory mean that HA Memgraph will lose all data upon system failure or restart?
+
+No. HA Memgraph uses RAM as the primary storage for data. However, HA Memgraph
+needs to replicate each change on the majority of the cluster and implicitly
+persists data to disk.  On restart, from the machine perspective, HA Memgraph
+uses snapshot created from the previous run, if any, and persisted Raft logs to
+recover its state to what it was before shutting down. After re-joining the
+cluster, additional syncing takes place to ensure data integrity.
+
+From the user perspective, a leader failover will occur on system failure or
+restart, and the user will not be affected with the failure, and the state of
+the data will stay the same as long as a majority of the cluster is alive.
+
+3. ##### How far have you scaled your system?
+
+It depends on the dataset and the queries, in other words, the workload. HA
+Memgraph scales up to the size of the main memory or disk space where properties
+and Raft logs could be stored on disk. In a distributed system, the graph is
```

Comment:
Shouldn't we remove the distributed system part.
I would also mention that write performance drops with addition of nodes in a Raft cluster and that we recommend odd-sized clusters...